apps/system/js/permission_manager.js

```diff
@@ -75,6 +75,7 @@
       window.addEventListener('attentionopening', this);
       window.addEventListener('attentionopened', this);
       window.addEventListener('lockscreen-appopened', this);
+      window.addEventListener('screenchange', this);
```

Comment:
let's call discard to clean the state in gecko.
window.addEventListener('screenchange', this.discardPermissionRequest);